client/app/puregetters/requests.coffee

```diff
@@ -0,0 +1,106 @@
+{RequestStatus, Requests} = require '../constants/app_constants'
+
+module.exports =
+    getRequests: (state) ->
+        state.get('requests')
+
+    get: (state, req) ->
+        @getRequests(state).get req
+
+    isLoading: (state, req) ->
+        RequestStatus.INFLIGHT is @get(state, req)?.status
+
+    getRefreshes: (state) ->
+        state.get('refreshes')
+
+    isRefreshError: (state) ->
+        @getRefreshes(state).get('errors')?.length
+
+    isRefreshing: (state)->
+        @getRefreshes(state).size isnt 0 or
+        @isLoading(state, Requests.REFRESH_MAILBOX)
+
+    isIndexing: (state, accountID) ->
```

Comment:
Is this still relevant? Why comment all this method?
Can you add some comments to explain all this (according to our discussions during poker planning) please?

I think there is a misunderstanding into the implementation (because of my work mainly, sorry) between : import and indexing. Add comment may be a solution to explain that and how to change it.